pkg/test/expectations/expectations.go

```diff
@@ -336,7 +428,13 @@ func ExpectManualBindingWithOffset(offset int, ctx context.Context, c client.Cli
 		Target: v1.ObjectReference{
 			Name: node.Name,
 		},
-	})).To(Succeed())
+	})
+	// This happens when we are scheduling deleting pods. The new pod will eventually go on the new node,
+	// but we are currently modeling the pod that exists on the deleting node
+	if errors.IsConflict(err) {
```

Comment:
Not sure I understand the need for this from the comment, unless this is a problem that has existing and it's being fixed now. Why would this happen now with Machines instead of nodes especially if we're binding them ourselves in the test package?

Reply:
Good catch, there was a lot of back and forth on this codepath, but you're correct that this doesn't need to exist anymore.